clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerProtocolTest.java

```diff
@@ -56,7 +56,7 @@ public void serializeDeserializeSubscriptionAllVersions() {
             new TopicPartition("foo", 0),
             new TopicPartition("bar", 0));
         Subscription subscription = new Subscription(Arrays.asList("foo", "bar"),
-            ByteBuffer.wrap("hello".getBytes()), ownedPartitions, generationId);
+            ByteBuffer.wrap("hello".getBytes()), ownedPartitions, generationId, Optional.empty());
```

Comment:
It would be great if we could extend this test and possibly others in the file to cover the protocol changes.

Reply:
Done.